app/views/catarse_bootstrap/layouts/catarse_bootstrap.html.slim

```diff
@@ -36,6 +36,7 @@ html
       = render_mixpanel_script
 
   body#catarse_bootstrap[data-controller-name=controller_name data-action=action_name data-user="#{current_user.try(:to_analytics_json)}" data-referral="#{session[:referral_link]}" data-id=params[:id] data-mobile=browser.mobile?.to_s itemscope itemtype="http://schema.org/WebPage" class="#{'body-alert' if can_display_pending_refund_alert?} #{content_for :body_class}"] 
+    script(async src="https://d335luupugsy2.cloudfront.net/js/loader-scripts/09f3866f-4c20-40a4-bfb8-68c2310d3964-loader.js" type="text/javascript")
```

Comment:
lets use a condition when we have CatarseSettings[:rd_token] to not load for everyone.